include/ultra64/thread.h

```diff
@@ -1,57 +1,65 @@
 #ifndef _ULTRA64_THREAD_H_
 #define _ULTRA64_THREAD_H_
 
-/* Types */
+#define OS_PRIORITY_IDLE        10
```

Comment:
I don't know if it was different in an older version of the SDK OoT uses (I doubt it), but on my OS2.0L file, this define reads:
```c
#define OS_PRIORITY_IDLE      0 /* Must be 0 */
```
It's probably more accurate to place this value in the game's code, not the OS's.

Reply:
I adjusted the priorities in the header with what Oot uses for each thread.
In this case:
https://github.com/zeldaret/oot/blob/5f3f065aa71ef75e28e5e9280657267bbdca24ff/src/boot/boot_main.c#L27-L29
But it's true that `ultra64/thread.h` is maybe not the best place to put these.

Reply:
Are we sure the "idle" thread and the "idle" priority are even related?

Reply:
Well yes, all games using the OS scheduler require an idle thread, and this is the value that's meant to be used.

But the OoT devs decided to go with 10 for no apparent reason. The idle thread usually should be given the smallest priority right after spawning some other thread (like main).

Reply:
Actually, they do end up setting it to 0 after spawning main after all.

If anything, OS_PRIORITY_IDLE should be used instead of the 0 constant, here: https://github.com/zeldaret/oot/blob/master/src/boot/idle.c#L86

So, it doesn't matter which priority level you choose for the initial creation of the idle thread then. For instance, I personally set it to 1, but it's probably fine even if it were 0 since there's literally no other thread besides idle at that point.